doc/dynamic-environments/quickstart.mkd

```diff
@@ -3,22 +3,21 @@
 This intent of this document is to serve as a basic guide for getting started with r10k and a fresh Puppet installation, including the following:
 
 * Installing Puppet and its dependencies
```

Comment:
How about changing two lines up to "getting started with r10k and the latest Puppet version", so we don't have to keep bumping a number?